README.md

```diff
@@ -38,12 +38,13 @@ for each call to `TracerObserver.onStart()`.
 
 ## Registering API extensions
 
-There are two ways an extension API can be registered for use with a `Tracer`.
+There are three ways an extension API can be registered for use with a `Tracer`.
 
 1) Native support within the `Tracer` implementation
 
 Some `Tracer` implementations may decide to implement support for the extension APIs directly, in which
```

Comment:
Brevity
>...  , in which case they will implement the `APIExtensionsManager` interface ...

can be simply
>... by implementing the `APIExtensionsManager` interface ...

(Maybe make the APIExtensionsManager a link to the interface itself?)